micro-ui/web/micro-ui-internals/packages/modules/workbench/src/hooks/inbox.js

```
@@ -0,0 +1,116 @@
+import { useQuery, useQueryClient } from "react-query";
+import axios from "axios";
+
+const useMDMSPopupSearch = (criteria) => {
+  const [moduleName, masterName] = criteria?.body?.MdmsCriteria?.schemaCode?.split(".");
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)